.github/workflows/clang-format.yml

```diff
@@ -10,9 +10,9 @@ jobs:
     steps:
     - name: Checkout Sources
       uses: actions/checkout@v1
-      
+
     - name: clang-format lint
       uses: DoozyX/clang-format-lint-action@v0.3.1
       with:
         # List of extensions to check
-        extensions: c,h
+        extensions: c
```

Comment:
Maybe generate format disables if needed for the generated files rather than suspend for all header files?

Reply:
Sadly, I don't control the generated code in those headers